src/lib/chat/matrix-client.ts

```diff
@@ -613,6 +613,44 @@ export class MatrixClient implements IChatClient {
     await this.matrix.redactEvent(roomId, messageId);
   }
 
+  async getRoomIdForAlias(alias: string): Promise<string | undefined> {
+    await this.waitForConnection();
+    return await this.matrix
+      .getRoomIdForAlias(alias) // { room_id, servers[] }
+      .catch((err) => {
+        if (err.errcode === 'M_NOT_FOUND') {
+          Promise.resolve(undefined);
+        }
+
+        if (err.errcode === 'M_INVALID_PARAM') {
+          // handle invalid alias error
```

Comment:
for us this should just be the same as if the room does not exist. There's no reason to tell the user anything else, I don't think.

Reply:
👍